src/h5web/providers/h5grove/h5grove-api.ts

```diff
@@ -74,19 +67,21 @@ export class H5GroveApi extends ProviderApi {
     }
 
     const { data } = await this.client.get<H5GroveAttrValuesResponse>(
-      `/attr/?${encodeQueryParams({ file: this.filepath, path })}`
+      `/attr/`,
+      { params: { path } }
     );
 
     this.attrValuesCache.set(path, data);
     return data;
   }
 
   private async fetchData(
-    params: ValueRequestParams
+    params: ValuesStoreParams
   ): Promise<H5GroveDataResponse> {
     const { data } = await this.cancellableFetchValue<H5GroveDataResponse>(
-      `/data/?${encodeQueryParams({ file: this.filepath, ...params })}`,
-      params
+      `/data/`,
+      params,
+      { ...params }
```

Comment:
You could let `queryParams` fall back to `storeParams` in `cancellableFetchValue` : `queryParams = storeParams`. This way you don't have to pass them twice here.